src/main/java/spoon/reflect/visitor/printer/change/SourceFragment.java

```diff
@@ -250,6 +250,8 @@ private SourceFragment addChild(SourceFragment parentFragment, CtElement otherEl
 								addChild(otherFragment);
 								return otherFragment;
 							}
+                            addChild(otherFragment);
```

Comment:
note: this line handles the case when spoon source positions are not consistent. I would prefer to try to fix source positions in spoon first. Only in case when it is not possible or there are some cases when position is inconsistent for some good reason, we should add such child using code you suggest here. But we should not do it now for all cases.

OK?

Reply:
Of course. I'm just using this as a workaround for my repair bot. Even after making this change, exceptions were thrown at other places so I just commented them and it worked. In the `CMP compare(SourceFragment)' function, when the two fragments overlaped, it can be checked which case the situation is closest to and return OTHER_IS_BEFORE/AFTER depending on it.